### PR TITLE
Portfolio Mobile-First 💻 

### PR DESCRIPTION
**Pull request by evansnyamekye**
- Created new works images-folder to enhance the cards display of the portfolio website 🖼️ . 
- I used HTML5 to build the entire portfolio card-works section🔧 .
- I used CSS to style and layout all the web pages contained in the mobile-first portfolio website ⚙️.
- Using CSS, I styled the required font, color, size, and spacing of the portfolio webpage 🧑‍💻 .
- I used Flexbox to properly align-items on the portfolio webpage 🔲 .
-
Thank you 🚀.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -120,9 +128,213 @@ body {
   opacity: 0.5;
 }
 
-@media (min-width: 786px) {
+/* Work  */
+.work {
```
Comment (amejid) [inline styles.css:143]: - [ ] You did a great job with the design. :+1: Let's work on more details to meet the project requirements. Kindly use `grid` for the container holding the entire cards and please use `flex` for the individual cards. That way it'll be easier when working on the desktop version of the page. 🥸️

**Requirements**
Screenshot from 2023-07-05 09-19-05

File: index.html [hunk 2]
```diff
@@ -55,8 +60,106 @@ <h1>I’m Nyamekye.
...
+                        </ul>
+                        <button class="btn-default">See project</button>
+                    </div>
+                    <div class="card-left-desk">
+                        <img src="images/work/Snapshoot Portfolio(3).png" alt="snap-card">
+                    </div>
+                </div>
+            </div>
+        </section>
+        <!-- end of works -->
```
Comment (amejid) [inline index.html:161]: - [ ] Kindly note that the `About` section is part of this milestone. I highly suggest you work on it. Please also make sure to align the contents of the About section with flexbox. 🤝️

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @evansnyamekye, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Good UI design :heavy_check_mark: 
- Nice semantic structure :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @evansnyamekye, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Flexbox used ✔️
- Grid used ✔️
- Required changes fixed ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**